airframe-http-okhttp/src/test/scala/wvlet/airframe/http/okhttp/OkHttpClientTest.scala

```diff
@@ -77,6 +79,12 @@ trait FinagleClientTestApi extends LogSupport {
     Response(Status.Forbidden)
   }
 
+  @Endpoint(method = HttpMethod.GET, path = "/readtimeout")
+  def readTimeout: String = {
+    Thread.sleep(3000)
```

Comment:
ok. So it doesn't mean we always wait 3 sec x 2 (with retry) because timeout is configured to 10ms. Good